README.md

```diff
@@ -82,9 +82,9 @@ The `director` directory contains the Django application. In most instances you
 `make` commands:
 
 ```bash
-make director-venv  # set up the virtual environment and install requirements
-make director-create-devdb  # populate the dev DB with test accounts (usernames == passwords)
-make director-run  # start the Django server
+make setup  # set up the virtual environment and install requirements
+make -C director create-devdb  # populate the dev DB with test accounts (usernames == passwords)
```

Comment:
Wasn't sure if this was something we wanted to make an alias for in the top level Makefile or not. Felt better to leave it to the director, but let me know and I can change it.